samples/config_helper_vulkan.cc

```diff
@@ -609,6 +612,27 @@ std::string deviceTypeToName(VkPhysicalDeviceType type) {
   return "unknown";
 }
 
+std::string stageFlagBitsToNames(const VkShaderStageFlags bits) {
+
+    std::string result;
```

Comment:
This should be an `std::ostringstream` and move the declaration down to line 626

Reply:
done